PSCompatibilityCollector/Microsoft.PowerShell.CrossCompatibility/Microsoft.PowerShell.CrossCompatibility.csproj

```
@@ -12,14 +12,14 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
     <!--  The version of Newtonsoft.Json needs to be newer than the version in the oldest supported version of PowerShell 7: https://github.com/PowerShell/PowerShell/blob/v7.0.11/src/System.Management.Automation/System.Management.Automation.csproj#L15 -->
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
```

Now that PowerShell 7.0 is out of support, we bump the version used by PowerShell Core to 13.0.1, which is the version used by the latest version of 7.2
https://github.com/PowerShell/PowerShell/blob/978d190594161a74c79e040eff3fdce8ddcff708/src/System.Management.Automation/System.Management.Automation.csproj#LL15C8-L15C8
```suggestion
    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```